SIT.Manager.Avalonia/ViewModels/PlayPageViewModel.cs

```diff
@@ -342,14 +341,14 @@ private async Task ConnectToServer(bool launchServer = false)
         string backendUrl = serverAddress.AbsoluteUri[..^(SITVersion >= standardUriFormatSupportedVersion ? 0 : 1)];
 
         // Launch game
-        string launchArguments = CreateLaunchArugments(new TarkovLaunchConfig { BackendUrl = backendUrl }, token);
+        string launchArguments = CreateLaunchArguments(new TarkovLaunchConfig { BackendUrl = backendUrl }, token);
         try
         {
             _tarkovClientService.Start(launchArguments);
         }
         catch (Exception ex)
         {
-            _logger.LogError("An exception occured while launching Tarkov: {exMessage}", ex.Message);
+            _logger.LogError($"An exception occured while launching Tarkov: {ex.Message}");
```

Comment:
This is the wrong format for the log. The compiler will actually get grumpy at you if you do this

Reply:
Sorry, will bring it back